### PR TITLE
JohnZ/enableBCC

### LINKED ISSUE #706: Enable BCC Only
image

Emails should only BCC when sending

### PR DESCRIPTION
**Pull request by JohnZ9865**
BCC is now removed as a feature, they now recommend doing it through personalization (as we talked about last time), so I did that.

image

### CODE REVIEW

**Comment by JohnZ9865** [comment]
Formatted email and the array looks like this: image

Which I think follows twilio guideline of this: https://docs.sendgrid.com/for-developers/sending-email/personalizations
image

File: src/app/admin/messenger/page.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const MessengerPage = () => {
         committees: 1,
       }}
     >
+      {console.log("CONSLOE.LOG: hereeee")}
```
Comment (shahdivyank) [inline src/app/admin/messenger/page.js]: remove console log

File: src/app/api/messenger/route.js [hunk 2]
```diff
@@ -19,6 +20,7 @@ export async function PUT(req) {
     );
   }
 
+  console.log("we ever here???");
```
Comment (shahdivyank) [inline src/app/api/messenger/route.js]: remove console log

Comment (JohnZ9865) [reply src/app/api/messenger/route.js]: did it

**Comment by shahdivyank** [comment]
Dont need since we changed over to SendinBlue (Brevo) :sob:

**Pull request closed without merging**